assets/controller.yaml

```diff
@@ -228,6 +231,7 @@ spec:
           - --upstream=http://127.0.0.1:8204/
           - --tls-cert-file=/etc/tls/private/tls.crt
           - --tls-private-key-file=/etc/tls/private/tls.key
+          - --tls-cipher-suites=${TLS_CIPHER_SUITES}
```

Comment:
iirc - this was an issue with openshift client connecting to vcenter too. We are not going to fix that?

Reply:
Do we have a bug for that somewhere? I don't mind picking it up, I just haven't seen it mentioned before.

Reply:
Also, if this is correct, it looks like they already dropped support for DES ciphers in vSphere 5.x:
https://kb.vmware.com/s/article/1018510
So I don't think we would have to make any client-side changes, but maybe I'm missing something.